redbot/core/commands/commands.py

```diff
@@ -530,7 +551,15 @@ async def can_see(self, ctx: "Context"):
         return True
 
     def disable_in(self, guild: discord.Guild) -> bool:
-        """Disable this command in the given guild.
+        """
+        Disable this command in the given guild.
+
+        This is generally called by the settings managed with
+        the ``[p]command disable global/server`` commands and
+        any changes made outside of that will not persist after cog
```

Comment:
```suggestion
        the ``[p]command disable global/server`` commands.
        Any changes made outside of that will not persist after cog
```
Sort of a run on, but should this even be public if this doesn't behave as expected without support from the cog?

Reply:
No, but we can't remove it until 3.6 and we should probably also provide alternative, safe APIs for command disabling in such cases. We should track it as an issue.